Test/Graph/test_graph.py

```diff
@@ -73,3 +82,6 @@ def test_computes_distances():
             for i in range(graph.n_sites):
                 for j in range(graph.n_sites):
                     assert d1[i][j] == d[i][j]
+
+
+test_edges_are_correct()
```

Comment:
We probably shoudn't call `test_*` functions explicitly. `pytest` runs them automatically.

Reply:
Indeed that's a leftover